shared/src/commonMain/kotlin/org/hyperskill/app/home/presentation/HomeActionDispatcher.kt

```diff
@@ -137,6 +107,30 @@ class HomeActionDispatcher(
         }
     }
 
+    private suspend fun fetchHomeScreenData() {
```

Comment:
Let's use the same style for action handlers helper functions - if necessary we provide action and onNewMessage block.

See for example `ProgressScreenActionDispatcher.handleFetchTrackWithProgressAction` and `ProjectSelectionListActionDispatcher.handleFetchContentAction`